lib/ui/src/components/preview/preview.tsx

```diff
@@ -135,11 +135,13 @@ const Preview: FunctionComponent<PreviewProps> = (props) => {
     baseUrl,
     withLoader = true,
   } = props;
-  const { isToolshown } = options;
   const { getElements } = api;
 
   const tabs = useTabs(previewId, baseUrl, withLoader, getElements, story);
 
+  const isToolshown =
+    !(viewMode === 'docs' && tabs.filter((t) => !t.hidden).length < 2) && options.isToolshown;
```

Comment:
When the viewMode is docs, we only show tabs
When the number of tabs is 1, there's no point in showing them
When we don't show anything, the toolbar should be collapsed

AFAIK this was exactly the issue in #10592

This code makes it so when there's only 1 tab shown in docsmode the toolbar collapses

Reply:
I didn't repro #10592 but @hipstersmoothie 's issue is that "nothing is rendered". AFAICT this change renders even less than what was there before, so I don't understand how this could solve that.

Reply:
I can set up an example repo if neeed

Reply:
Best is repro in official-storybook

Reply:
@ndelangen @shilman I've ran this locally and the only issue I see is that on initial load,
the MDX `docs-only` page that I select on `SET_STORIES` shows the toolbar like:

![image](https://user-images.githubusercontent.com/260185/83985222-ecb99900-a8fd-11ea-8e30-97b47c668fd2.png)

and after navigate another story with Canvas/Docs tab, and going back, it's not shown:

![image](https://user-images.githubusercontent.com/260185/83985270-24c0dc00-a8fe-11ea-8ea5-349ca522534c.png)

the tabs/reordering seems to work fine with `beta.22` :D
now I'd like to know how to set the `selectedTab` in a similar way as `selectedPanel` hehe